doc/api/util.md

```diff
@@ -257,6 +257,30 @@ intended as a debugging tool. Some input values can have a significant
 performance overhead that can block the event loop. Use this function
 with care and never in a hot code path.
 
+## util.hasColors([stream, [env]])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `stream` {Stream} A [Stream][]. Defaults to `process.stdout`.
+* `env` {object} A object containing the environment variables to check.
+  Defaults to `process.env`.
+* Returns: {number}
+
+Returns:
+* 0 for no,
+* 1 for 16,
+* 2 for 256,
+* 3 for 16,777,216
+colors supported.
```

Comment:
Perhaps just make the return the actual values... e.g. return `0`, `16`, `256`, or `16777216`

Reply:
Hm, I see your point but I personally think it is nicer to stick to the levels.